src/ngraph_utils.cc

```diff
@@ -378,7 +378,18 @@ void AllreduceOpControlOrder(
       }
     }
   }
-};
+}
+
+bool IsProcessedByNgraphPass(Graph* g) {
```

Comment:
This addition is the main change wrt the old PR.

Please note where its called to disable ngraphs rewrite pass